server/form-pages/apply/area-and-funding/funding-information/index.ts

```diff
@@ -0,0 +1,11 @@
+/* istanbul ignore file */
+
+import { Task } from '../../../utils/decorators'
+import FundingInformationPage from './fundingInformation'
+
+@Task({
+  name: 'Add funding information',
+  slug: 'funding-information',
```

Comment:
question: do you think there is another `slug` value we could give the page (`fundingInformation.ts`), so it's different from the task's slug here? `funding-source` maybe? or `how-will-they-pay`?

Reply:
yes, good spot! I'm happy with `funding-source`